packages/format/__tests__/format-utils/bitcoin-format-utils.spec.ts

```diff
@@ -0,0 +1,73 @@
+import chai from 'chai';
+
+import { roundBTC, toBTC, toSats } from '../../lib/bitcoin';
+
+const expect = chai.expect;
+
+describe('Bitcoin formatting utilities', () => {
+  describe('toBTC', () => {
+    it('should correctly convert a single satoshi to BTC', () => {
```

Comment:
might be good to add a test for the case where non-number characters are provided

(note syntax has changed for chai testing errors being thrown: https://github.com/domenic/chai-as-promised/issues/173#issuecomment-588857915)

Reply:
I believe that's only the case where it's an error thrown from a Promise rejection.
Non-async errors still use .throw